frontend/model/state.js

```diff
@@ -510,6 +525,44 @@ const getters = {
       return 1
     }
     return 7
+  },
+  currentChatRoomId (state, getters) {
+    return state.currentChatRoomIDs[state.currentGroupId] || null
+  },
+  isPublicChatRoom (state, getters) {
+    return (chatRoomId: string) => {
+      return state[chatRoomId]?.attributes.privacyLevel !== CHATROOM_PRIVACY_LEVEL.PRIVATE
```

Comment:
Shouldn't this be:

```js
      return state[chatRoomId]?.attributes.privacyLevel === CHATROOM_PRIVACY_LEVEL.PUBLIC
```

?

Reply:
I guess there would be confusion in name of this getter.
We use this getter in order to check if we can access to the group without joining.
So even though they are public through all the users or users inside the group, users can access and view the messages and join by themselves.

Reply:
Used `isPrivateChatRoom` instead of `isPublicChatRoom`.

```
isPrivateChatRoom (state, getters) {
    return (chatRoomId: string) => {
      return state[chatRoomId]?.attributes.privacyLevel === CHATROOM_PRIVACY_LEVEL.PRIVATE
    }
}
```

I think this could solve the problem regarding naming confusion.